### PR TITLE
Enhancing UI for inviting

### LINKED ISSUE #2189: Enhancing UI for the invitation page.
**Is your feature request related to a problem? Please describe.**
Currently, the UI of the invite modal sheet appears basic and lacks a visually appealing design.

**Describe the solution you'd like**
I would like to enhance the UI of the invitation modal sheet to provide a more better user-friendly experience. This could include proper font text, intuitive design elements, and an overall polished look.

**Additional context**
Current look of the invite sheet :

Screenshot_1701450473

### PR DESCRIPTION
**Pull request by Dante291**
**What kind of change does this PR introduce?**

Enhancing UI for inviting other users to join an organization using QR.

**Issue Number:**

Fixes #2189 

**Did you add tests for your changes?**

Yes

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by Dante291** [comment]
@palisadoes Sorry for closing PR I was having some trouble to resolve conflicts and with formatting, this one is after latest pull could you please assign reviewers.

**Comment by Dante291** [comment]
Before: 

Screenshot_1701450473

after:

Screenshot_1701880428

**Comment by palisadoes** [comment]
Please try to get your code coverage up to 100% on the edited file

File: lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart [hunk 1]
```diff
@@ -169,95 +172,60 @@ class ProfilePageViewModel extends BaseModel {
...
               mainAxisAlignment: MainAxisAlignment.center,
               crossAxisAlignment: CrossAxisAlignment.center,
               children: [
+                iconButton(
+                  CustomPaint(
+                    size: const Size(48, 48 * 1),
+                    painter: AppLogo(),
+                  ),
+                  () {},
+                ),
```
Comment (noman2002) [inline lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart:202]: Please write the test for these lines.

**Review by noman2002 — CHANGES_REQUESTED** [review]

**Comment by Dante291** [comment]
Translation is working fine:
Screenshot_1702485306

**Comment by Dante291** [comment]
@palisadoes @noman2002 I added test for every method except logout because @Azad99-9 has did some modification to that method and also have written test for it in this [PR.](https://github.com/PalisadoesFoundation/talawa/pull/2227)

**Review by noman2002 — APPROVED** [review]
Looks good, Thank you.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**